vips/foreign.c

```diff
@@ -56,11 +57,22 @@ int save_png_buffer(VipsImage *in, void **buf, size_t *len, int strip, int compr
 // https://github.com/libvips/libvips/blob/master/libvips/foreign/webpsave.c#L524
 // https://libvips.github.io/libvips/API/current/VipsForeignSave.html#vips-webpsave-buffer
 int save_webp_buffer(VipsImage *in, void **buf, size_t *len, int strip, int quality, int lossless, int effort) {
+	const char* profile = "none";
```

Comment:
I think that setting the profile should happen higher up, we lose control when it's down here.